app/views/registerBusinessRepresenting.scala.html

```diff
@@ -35,14 +35,38 @@
 
         @components.heading("registerBusinessRepresenting.heading")
 
+        <div role="note" aria-label="Information" class="application-notice info-notice">
+            <p>The details you enter must match the exact format of their registered EORI address</p>
+        </div>
+
+        @components.input_text(
+            field = form("eoriNumber"),
+            label = messages("registerBusinessRepresenting.eoriNumber")
+        )
+
+        @components.input_text(
+            field = form("businessName"),
+            label = messages("registerBusinessRepresenting.businessName")
+        )
+
+        @components.input_text(
+            field = form("addressLine1"),
```

Comment:
Why `addressLine1`
Looks like we could call it `address`

Reply:
I would agree in this one, but I've just moved the same info we had on the first one (register EORI). I think this one needs to be align with our code where we have AdressLine1, AdressLine2 and AdressLine3. Maybe we need to change our EORI Details structure in order to sync with the real live data.